src/plugins/console/server/lib/spec_definitions/js/aggregations.ts

```diff
@@ -320,13 +320,40 @@ const rules = {
     geohash_grid: {
       __template: {
         field: '',
-        precision: 3,
+        precision: 5,
       },
       field: '{field}',
       precision: { __one_of: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12] },
       size: 10,
       shard_size: 10,
     },
+    geohex_grid: {
+      __template: {
+        field: '',
+        precision: 6,
+      },
+      field: '{field}',
+      precision: {
+        __one_of: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15],
+      },
+      size: 10,
+      shard_size: 10,
+    },
+    geotile_grid: {
+      __template: {
+        field: '',
+        precision: 7,
+      },
+      field: '{field}',
+      precision: {
+        __one_of: [
+          0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
+          25, 26, 27, 28, 29,
+        ],
+      },
+      size: 10,
```

Comment:
Is this a good default value? If I'm reading the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-geotilegrid-aggregation.html#_options_4) correctly, the default is 10,000.

Reply:
Yeah, at first I wasn't sure if we should use the 10,000 default, but I started to worry that parsing up to 10,000 hits in the results might be overwhelming for the browser and maybe unnecessary for a user who may only want to preview the results.

Reply:
Hmm, maybe I should make this `size: 10000` but just have `size: 10` in the `__template`?